### PR TITLE
Doc RUC

### PR DESCRIPTION
**Pull request by Oxicode**
Doc https://github.com/FakerPHP/Faker/pull/244

### CODE REVIEW

File: docs/locales/es_PE.md [hunk 1]
```diff
@@ -6,3 +6,24 @@
 // Generates a Peruvian Documento Nacional de Identidad (DNI) number
 echo $faker->dni; // '83367512'
 ```
+
+### `Faker\Provider\es_PE\Company`
+
+```php
+// Generates a Peruvian REG. UNICO DE CONTRIBUYENTES (RUC) number
+echo $faker->ruc; // '20105895410'
```
Comment (pimjansen) [inline docs/locales/es_PE.md:14]: Code indicates a 10x number

File: docs/locales/es_PE.md [hunk 2]
```diff
@@ -6,3 +6,24 @@
...
+```
+
+### `Faker\Provider\es_PE\Company`
+
+```php
+// Generates a Peruvian REG. UNICO DE CONTRIBUYENTES (RUC) number
+echo $faker->ruc(true); // '20105895410'
```
Comment (pimjansen) [inline docs/locales/es_PE.md:21]: Same here if im correct

**Review by pimjansen — CHANGES_REQUESTED** [review]

File: docs/locales/es_PE.md (hunk 2 above)
Comment (Oxicode) [reply docs/locales/es_PE.md]: done

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
Thanks

**Pull request merged**